root/edit/components/IntentionallyRawIcon.js

```diff
@@ -12,13 +12,13 @@ import * as React from 'react';
 import InformationIcon
   from '../../static/scripts/edit/components/InformationIcon';
 
-const IntentionallyUntranslatedIcon =
+const IntentionallyRawIcon =
   (): React.Element<typeof InformationIcon> => (
     <InformationIcon
       className="align-top"
-      title={l(`This field is intentionally left untranslated
-                (i.e. just as entered) to facilitate edit review`)}
+      title={l(`This field is left as it was originally entered
+                (untranslated, unformatted) to facilitate edit review`)}
```

Comment:
IMHO the end of the sentence is not useful (but to developers?), the sentence should be stopped still.

```suggestion
      title={l(`This field is left as it was originally entered
                (untranslated, unformatted).`)}
```

Reply:
Hm, I thought that was the point of the string. Since the user doesn't need a tooltip to see that it's untranslated & unformatted, just that it might not be obvious why (i.e. it's not a bug). +1 to the period!

Reply:
Added the period. I also feel that the end of the sentence is the whole point of why we do this and seems sensible to mention :/

Reply:
It’s obviously for edit review as it shows up for edit display only, and writing the obvious is unneeded. :wink:
Maybe just rephrase using _intentionally_ instead? e.g. “This field is intentionally shown as…).”?

Reply:
I still think that's a worse string but I really want to merge this today, so whatever, let's do that.

Reply:
Well, phrasing is not really my specialty either…